javaparser-core/src/main/java/com/github/javaparser/ast/CompilationUnit.java

```diff
@@ -500,6 +504,15 @@ public AnnotationDeclaration addAnnotationDeclaration(String name, Modifier.Keyw
     public Optional<ClassOrInterfaceDeclaration> getClassByName(String className) {
         return getTypes().stream().filter(type -> type.getNameAsString().equals(className) && type instanceof ClassOrInterfaceDeclaration && !((ClassOrInterfaceDeclaration) type).isInterface()).findFirst().map(t -> (ClassOrInterfaceDeclaration) t);
     }
+    
+    /**
+     * Try to get a locally class declaration by its name (top level or inner class)
```

Comment:
Typo - "get a local class declaration" (alternative: "get a locally declared class")

Reply:
Thank you for these corrections because my level in English is relatively low.